demos/notebooks/demo_online_cnmfE.ipynb

```diff
@@ -102,40 +131,51 @@
     "#%% start a cluster for parallel processing (if a cluster already exists it will be closed and a new session will be opened)\n",
     "if 'dview' in locals():\n",
     "    cm.stop_server(dview=dview)\n",
-    "c, dview, n_processes = cm.cluster.setup_cluster(\n",
-    "    backend='multiprocessing', n_processes=None, single_thread=False)"
+    "c, dview, n_processes = cm.cluster.setup_cluster(backend='multiprocessing', \n",
+    "                                                 n_processes=None,  # None\n",
+    "                                                 single_thread=False)  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Initialize motion correction estimator with the parameters, and run to fit data:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "mc = MotionCorrect(fnames, dview=dview, **opts.get_group('motion'))\n",
+    "%%time\n",
+    "mc = MotionCorrect(fnames, dview=dview, **opts.get_group('motion')) \n",
     "mc.motion_correct(save_movie=True)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### inspect motion correction results"
+    "### inspect motion correction results\n",
+    "This will load into memory so can take a while."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "inspect_results = False\n",
+    "inspect_results = True\n",
     "if inspect_results:\n",
-    "    cm.concatenate((cm.load(fnames), cm.load(mc.mmap_file)), axis=1).play()\n",
+    "    cm.concatenate((cm.load(fnames).resize(1, 1, temporal_downsampling), \n",
+    "                    cm.load(mc.mmap_file).resize(1, 1, temporal_downsampling)), axis=2).play(fr=30,\n",
+    "                                                         magnification=1,\n",
+    "                                                         gain=0.6,\n",
+    "                                                         plot_text=True,\n",
+    "                                                         do_loop=True)\n",
```

Comment:
if we're going to default to looping, we should tell the user that they shouldn't keep waiting

Reply:
Good point I'll cut that.